services/tmdb/index.js

```diff
@@ -0,0 +1,29 @@
+const axios = require('axios');
+const { requests } = require('../../utils');
+
```

Comment:
JDoc to specify the return

```suggestion
/**
 * Requests configs?
 * @return  {??}  ??
 */
```